src/pages/research/index.tsx

```diff
@@ -1,14 +1,467 @@
 import React from "react";
-import classnames from "classnames";
 import Head from "next/head";
+import Accordion from "react-bootstrap/Accordion";
+import Link from "next/link";
 
 import { PageBanner } from "../../components/Banner";
 import { BannerCTA } from "../../components/CallToActionBanner";
-import { Heading } from "../../components/Heading";
-import { ResearchCard } from "../../components/ResearchCard";
-import Stack from "../../components/Stack";
 
-import styles from "./index.module.scss";
```

Comment:
Given that this is no longer imported, can you delete the adjacent `index.module.scss` file?